micrometer/src/main/java/feign/micrometer/FeignObservationConvention.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2012-2022 The Feign Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package feign.micrometer;
+
+import io.micrometer.core.instrument.binder.httpcomponents.DefaultApacheHttpClientObservationConvention;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationConvention;
+
+/**
+ * {@link ObservationConvention} for Feign.
+ *
+ * @since 1.10.0
+ * @see DefaultApacheHttpClientObservationConvention
```

Comment:
Why this `@see` ? Also `@since 1.10.0` seems like a copy from micrometer ;)

Reply:
I'm not sure either, you wrote that =)